nupic/__init__.py

```diff
@@ -19,3 +19,7 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 __version__ = "0.3.0.dev0"
+
+import os
+NUPIC_ROOT = os.environ.get('NUPIC', os.path.dirname(os.path.realpath(__file__)))
```

Comment:
Hi David, my change will consider `$NUPIC` if it is set or consider the path from `__file__` as you suggested. I didn't remove `$NUPIC` altogether because I wanted to avoid side-effects. If you guys are sure nobody relies on it, I'm more than happy to remove it for good. :+1: